test/Quoter.t.sol

```diff
@@ -0,0 +1,441 @@
+//SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.20;
+
+import "forge-std/console.sol";
+import {Test} from "forge-std/Test.sol";
+import "../contracts/libraries/SwapIntention.sol";
+import {Quoter} from "../contracts/lens/Quoter.sol";
+import {LiquidityAmounts} from "../contracts/libraries/LiquidityAmounts.sol";
+import {MockERC20} from "solmate/test/utils/mocks/MockERC20.sol";
+import {BalanceDelta} from "@uniswap/v4-core/contracts/types/BalanceDelta.sol";
+import {SafeCast} from "@uniswap/v4-core/contracts/libraries/SafeCast.sol";
+import {Deployers} from "@uniswap/v4-core/test/foundry-tests/utils/Deployers.sol";
+import {IHooks} from "@uniswap/v4-core/contracts/interfaces/IHooks.sol";
+import {PoolModifyPositionTest} from "@uniswap/v4-core/contracts/test/PoolModifyPositionTest.sol";
+import {PoolId, PoolIdLibrary} from "@uniswap/v4-core/contracts/types/PoolId.sol";
+import {PoolKey} from "@uniswap/v4-core/contracts/types/PoolKey.sol";
+import {PoolManager} from "@uniswap/v4-core/contracts/PoolManager.sol";
+import {IPoolManager} from "@uniswap/v4-core/contracts/interfaces/IPoolManager.sol";
+import {Currency, CurrencyLibrary} from "@uniswap/v4-core/contracts/types/Currency.sol";
+import {TickMath} from "@uniswap/v4-core/contracts/libraries/TickMath.sol";
+
+contract QuoterTest is Test, Deployers {
+    using SafeCast for *;
+    using PoolIdLibrary for PoolKey;
+
+    // Min tick for full range with tick spacing of 60
+    int24 internal constant MIN_TICK = -887220;
```

Comment:
nit: import from TickMath

Reply:
tickMath only defines constants for tick spacing of 1 I think